pkg/models/content.go

```diff
@@ -0,0 +1,21 @@
+package models
+
+import (
+	"openpitrix.io/logger"
+
+	"openpitrix.io/notification/pkg/util/jsonutil"
+)
+
+type Content map[string]string
+
+func DecodeContent(data string) (*Content, error) {
+	logger.Infof(nil, "DecodeContent data=[%s]", data)
+
+	content := new(Content)
+	err := jsonutil.Decode([]byte(data), content)
+	if err != nil {
+		logger.Warnf(nil, "Try to decode as format[{\"content_type\": \"html\",\"content\": \"xxxxxxxxxxxx\"}], decode [%s] into content info failed: %+v", data, err)
```

Comment:
The log maybe not exactly。
Try to decode as format {\"content_type\": \"content\"}